package.json

```diff
@@ -9,7 +9,7 @@
     "@types/node": "20.6.0",
     "@types/react": "18.2.21",
     "@types/react-calendar-heatmap": "^1.6.3",
-    "prettier": "3.0.3",
+    "prettier": "^3.0.3",
```

Comment:
Why do we need this change?
Please avoid changing files not related to your task :smile: